crates/gitql-ast/src/function.rs

```diff
@@ -24,6 +24,9 @@ lazy_static! {
         map.insert("ltrim", text_left_trim);
         map.insert("rtrim", text_right_trim);
         map.insert("len", text_len);
+        map.insert("ascii", text_ascii);
```

Comment:
Please remove unreleated code to `DATALENGTH` function

Make sure to create new branch from the master not from last branch

```
git checkout master
git checkout -b new_branch_name
```